opensuse-kubectl-docker/Dockerfile

```diff
@@ -19,8 +17,11 @@ RUN true \
     curl \
     wget \
     tar \
-    helm \
     direnv \
+    aaa_base bash-completion docker-bash-completion systemd-bash-completion \
```

Comment:
bash completion is set up in `/etc/bash.bashrc`, which is in the aaa_base package.

We can instead:

1. `source /etc/profile.d/bash_completion.sh` in ~/.bashrc
2. Get rid of bash completion altogether. Since this container is meant for interactive usage, I'd rather not do this.